rest-api/calamari_rest/serializers/v2.py

```diff
@@ -142,6 +142,8 @@ class Meta:
     public_addr = serializers.CharField(read_only=True, help_text="Public/frontend IP address")
     cluster_addr = serializers.CharField(read_only=True, help_text="Cluster/backend IP address")
     crush_node_ancestry = serializers.Field(help_text="An ordered list of CRUSH node ids that represent a path from the parent node of this OSD up to the root of the tree")
+    backend_partition_path = serializers.CharField(read_only=True, help_text="Full path to the storage partition targeted by this OSD")
```

Comment:
None. This set of changes is only to address information about data partitions and devices (hence the "backend" modifier prepended to both new elements, leaving room for a "journal" version). We can make a set of changes around journals if we decide it's appropriate (which, at some point, I expect to do). It'll just involve a call to the function I created to pull the data device information, but targeted at the journal device instead. I'd rather focus on creating the entire set of information and capabilities for the data devices first, personally.